### PR TITLE
Add link to Pixhawk 6X-RT doc page

### PR DESCRIPTION
**Pull request by dk7xe**
Add Pixhawk 6X-RT doc link to the list of Flight controllers.
@hamishwillee have forgotten adding the link in previous pull request.

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -168,6 +168,7 @@
       * [CUAV Pixhawk V6X (FMUv6X)](flight_controller/cuav_pixhawk_v6x.md)
       * [Holybro Pixhawk 6X-RT (FMUv6X-RT)](flight_controller/pixhawk6x-rt.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:169]: See, I added it  here.

File: en/SUMMARY.md [hunk 2]
```diff
@@ -168,6 +168,7 @@
...
       * [Holybro Pixhawk 6X (FMUv6X)](flight_controller/pixhawk6x.md)
+      * [Holybro Pixhawk 6X-RT (FMUv6X-RT)](flight_controller/pixhawk6x-rt.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:171]: I added it two above

File: en/flight_controller/README.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ You can also try PX4 on a [Complete Vehicle](../complete_vehicles/README.md) (co
   - [Holybro Pixhawk 4 Mini (FMUv5)](../flight_controller/pixhawk4_mini.md)
   - [Holybro Pixhawk 5X (FMUv5X)](../flight_controller/pixhawk5x.md)
   - [Holybro Pixhawk 6X (FMUv6X)](../flight_controller/pixhawk6x.md)
+  - [Holybro Pixhawk 6X-RT (FMUv6X-RT)](../flight_controller/pixhawk6x-rt.md)
```
Comment (hamishwillee) [inline en/flight_controller/README.md:23]: This is good.

File: en/SUMMARY.md (hunk 2 above)
Comment (hamishwillee) [reply en/SUMMARY.md:171]:
```suggestion
```

File: en/flight_controller/pixhawk6x-rt.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ It is based on the [Pixhawk​​® Autopilot FMUv6X Standard](https://github.co
 
 Equipped with a high performance NXP i.mx RT1176 dual core Processor, modular design, triple redundancy, temperature-controlled IMU board, isolated sensor domains, delivering incredible performance, reliability, and flexibility.
 
-[Pixhawk6X-RT Upright Image](../../assets/flight_controller/pixhawk6x-rt/pixhawk6x-rt.png)
+<img src="../../assets/flight_controller/pixhawk6x-rt/pixhawk6x-rt.png" width="460px" title="Pixhawk6X-RT Upright Image" /> <img src="../../assets/flight_controller/pixhawk6x/pixhawk6x_exploded_diagram.png" width="400px" title="Pixhawk6X Exploded Image" />
```
Comment (hamishwillee) [inline en/flight_controller/pixhawk6x-rt.md:14]: These don't render side by side on the built version. I assume your intent is that?

```suggestion
<img src="../../assets/flight_controller/pixhawk6x-rt/pixhawk6x-rt.png" width="350px" title="Pixhawk6X-RT Upright Image" /> <img src="../../assets/flight_controller/pixhawk6x/pixhawk6x_exploded_diagram.png" width="300px" title="Pixhawk6X Exploded Image" />
```

File: en/flight_controller/pixhawk6x-rt.md [hunk 5]
```diff
@@ -198,9 +198,11 @@ It is pre-built and automatically installed by _QGroundControl_ when appropriate
 To [build PX4](../dev_setup/building_px4.md) for this target:
 
 ```
-make px4_fmu-v6x_default
+make nxp_fmurt1170-v1_default
```
Comment (hamishwillee) [inline en/flight_controller/pixhawk6x-rt.md:201]: Will this change back when PX4 SWS is enabled?

I think we should also add `git checkout upstream pr-imxrt1170-imxrt1062 # Upstream is PX4-Autopilot repo` above this.

File: en/flight_controller/pixhawk6x-rt.md [hunk 6]
```diff
@@ -198,9 +198,11 @@ It is pre-built and automatically installed by _QGroundControl_ when appropriate
...
 ```
-
+:::warning
+As of now PX4 SW enablement is done in px4 branch pr-imxrt1170-imxrt1062 only. Merge to PX4 main is expected soon.
+:::
```
Comment (hamishwillee) [inline en/flight_controller/pixhawk6x-rt.md:205]: Perhaps move this warning to the top of the page?

Comment (dk7xe) [reply en/flight_controller/pixhawk6x-rt.md:205]: Good idea. Will do on Monday.

Comment (hamishwillee) [reply en/flight_controller/pixhawk6x-rt.md:205]: Ping

File: en/flight_controller/pixhawk6x-rt.md (hunk 5 above)
Comment (hamishwillee) [reply en/flight_controller/pixhawk6x-rt.md:201]: Ping

File: en/flight_controller/pixhawk6x-rt.md (hunk 6 above)
Comment (dk7xe) [reply en/flight_controller/pixhawk6x-rt.md:205]: Agree, this should be at top of page. Anyhow the branch has changed again due to the target name change from nxp_fmu.. to px4_fmu-v6xrt. Please skip this.

File: en/flight_controller/pixhawk6x-rt.md (hunk 5 above)
Comment (dk7xe) [reply en/flight_controller/pixhawk6x-rt.md:201]: the final target name is now px4_fmu-v6xrt

File: en/flight_controller/pixhawk6x-rt.md (hunk 6 above)
Comment (dk7xe) [reply en/flight_controller/pixhawk6x-rt.md:205]: correction is done in https://github.com/NXPHoverGames/PX4-user_guide/pull/1

File: en/flight_controller/pixhawk6x-rt.md (hunk 5 above)
Comment (dk7xe) [reply en/flight_controller/pixhawk6x-rt.md:201]: correction is done in https://github.com/NXPHoverGames/PX4-user_guide/pull/1

Comment (hamishwillee) [reply en/flight_controller/pixhawk6x-rt.md:201]:
```suggestion
make px4_fmu-v6xrt
```

**Comment by hamishwillee** [comment]
@dk7xe Thanks. This wasn't quite right and I couldn't edit it (something in your permissions not quite right for github). So I have reproduced the relevant bits in #2853.

**Pull request closed without merging**